graphbolt/src/isin.cc

```diff
@@ -0,0 +1,45 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ *
+ * @file isin.cc
+ * @brief Isin op.
+ */
+
+#include <graphbolt/isin.h>
+
+namespace {
+static constexpr int kSearchGrainSize = 4096;
+}  // namespace
+
+namespace graphbolt {
+namespace sampling {
+
+torch::Tensor IsIn(
+    const torch::Tensor& elements, const torch::Tensor& test_elements) {
```

Comment:
Should add `elements.contiguous()` to guarantee `elements` is on consecutive memory.